.github/CODEOWNERS

```diff
@@ -1 +1 @@
-*	@gowrizrh @aligent/aligent-devops
+*	@kai-nguyen-aligent @aligent/aligent-devops
```

Comment:
I think this should be `*` to the microservices guild (and Kai) and then the .github directory for aligent devops

Reply:
@TheOrangePuff done :+1: